File: l001_Basics/l001Basics.cpp
#include <iostream>
using namespace std;

int mult(int a, int b)
{
    return a * b;
}

void add(int a, int b)
{
    int c = a + b;
    cout << c << endl;
}

int subtract(int a, int b)
{
    return a - b;
}

int divide(int a, int b)
{
    return a / b;
}

int main()
{
    cout << "Hello ";
    cout << "Arsh " << endl;
    // Agr 1 next line print krni hai to endl lgayege otherwise 1 hi line me aata jaega
    cout << "coder " << endl;

    // FUNCTIONS......
    int a, b; // Taking input from users
    cin >> a >> b;
    /*
        cout << a + b << endl;
        cout << a - b << endl;
        cout << a * b << endl;
        cout << a / b << endl;
        cout << a + b << endl; // Again + wale ka logic likhre hai, so better to make a function
    */

    int ans = mult(a, b); // return krra tha isliye 1 variable me catch krra, and usko print kra
    cout << ans << endl;

    add(a, b) ; // coz kuch return ni krra, direct call
    cout << divide(a, b) << endl; // 2nd way, agr kisi variable me catch nai krna toh
    cout << subtract(a, b) << endl;

    return 0;
}

File: l001_Basics/l001Basics.java
import java.util.Scanner;

public class l001Basics {

  public static Scanner scn = new Scanner(System.in);

  public static int mult(int a, int b) {
    return a * b;
  }

  public static void add(int a, int b) {
    int c = a + b;
    System.out.println(c);
  }

  public static int subtract(int a, int b) {
    return a - b;
  }

  public static int divide(int a, int b) {
    return a / b;
  }

  public static void main(String[] args) {
    System.out.print("Hello ");
    System.out.println("coder"); // ln -> is used for next line, jo bhi next op hoga vo next line se chalu hoga

    System.out.println("Arsh");

    // Functions
    System.out.println(mult(3, 2));
    System.out.println(subtract(3, 2));
    System.out.println(divide(3, 2));
    add(3, 2);
  }
}
